editoast/src/models/paced_train.rs

```diff
@@ -10,7 +10,7 @@ use editoast_schemas::train_schedule::Margins;
 use editoast_schemas::train_schedule::PathItem;
 use editoast_schemas::train_schedule::PowerRestrictionItem;
 use editoast_schemas::train_schedule::ScheduleItem;
-use editoast_schemas::train_schedule::TrainScheduleBase;
+use editoast_schemas::train_schedule::TrainSchedule as TrainScheduleSchema;
```

Comment:
nitpick/suggestion: Maybe we could do the same as with `model`, re-export all objects in the root level of `editoast_schemas`, so then you can use them as `editoast_schemas::TrainSchedule` 🤷 In particular, in files where both `models` and `schema` are used, they could be referred throughout the file as `editoast_schemas::TrainSchedule` and `models::TrainSchedule` (when there is a name that is used for 2 different things, I tend to prefer to explicitly prefix it inline everywhere, instead of `use`-ing it for one, and prefixing for the other).

Note that you could still `use editoast_schemas as schemas;` to shorten the prefix to only what we really need to disambiguate.

Reply:
(maybe we can even drop the `editoast_` prefix for crates? 👀 It brings very little benefits imo)

Reply:
We did that to match what other repos are doing like https://github.com/serde-rs/serde.

Reply:
Yeah I recall talking about that, but in the end since unlike `serde` or `diesel` we're not publishing our crates, it makes less sense to follow this practice imo (and it's cumbersome to write ~above all else~)